include/gatekeeper_gk.h

```diff
@@ -45,6 +45,13 @@ enum gk_flow_state { GK_REQUEST, GK_GRANTED, GK_DECLINED };
 struct gk_instance {
 	struct rte_hash   *ip_flow_hash_table;
 	struct flow_entry *ip_flow_entry_table;
+	/*
+	 * The timeout in seconds that the inactive
+	 * flow entry becomes expired.
+	 */
+	unsigned int      flow_timeout_sec;
+	/* Timeout for scanning one flow table bucket in ms. */
+	unsigned int      bucket_scan_timeout_ms;
```

Comment:
For ``bucket_scan_timeout_ms``, because we don't know the number of buckets ahead, we only specify the timeout for scanning the whole flow table ``flow_table_scan_timeout_ms``.